### ISSUE TITLE
Quiz about machine learning approaches

### ISSUE THREAD

**Issue by laureroche** [SEN1221TUD/Q2_2025#12]
Hi, 

I took a screenshot of this quiz question from the first lecture about machine learning approaches. I got the answers wrong, some of them are still not clear to me. Could you please provide me with further explanation about why each statement is True or False?

<img width="473" height="339" alt="Image" src="https://github.com/user-attachments/assets/1602aa7c-df50-46b8-9d2b-3c1a4676a0b8" />

Thank you in advance,

**Comment by sandervancranenburgh** [comment]
Sure.

Statement 1 is True. An ANN yields closed-form choice probabilities for each alternative, since probabilities are computed via explicit matrix operations and a softmax function, and therefore do not require numerical integration or simulation.

Statement 2 is False. ANNs do not, in general, exhibit the IIA property. In a fully connected network, the utility of an alternative may depend on the attributes of other alternatives, implying that the relative choice probabilities of two alternatives can be affected by the presence or attributes of additional alternatives.

Statement 3 is False. ANNs do not **inherently** violate utility maximisation principles. When used to represent systematic utilities and combined with a softmax choice rule, they can represent choice behaviour that is fully consistent with random utility maximisation.

Statement 4 is True. Unlike Mixed Logit models, standard ANNs rely exclusively on observed input features (attributes). As a result, individuals with identical observed characteristics receive identical utilities, meaning that genuinely unobserved taste heterogeneity across individuals is not captured.
